cmd/kubeadm/app/cmd/reset.go

```diff
@@ -68,15 +69,21 @@ func NewCmdReset(in io.Reader, out io.Writer) *cobra.Command {
 				kubeadmutil.CheckErr(err)
 			}
 
+			cfg, err := configutil.FetchInitConfigurationFromCluster(client, os.Stdout, "reset", false)
+			if err != nil {
+				cfg = nil
+				klog.Warningf("[reset] Unable to fetch the kubeadm-config ConfigMap from cluster: %v", err)
```

Comment:
we should just fail here, see:
https://github.com/kubernetes/kubernetes/blob/739af85d7a63dec41e07913c14c3fe26695804bb/cmd/kubeadm/app/util/config/cluster.go#L42

cfg is already `nil`, but also passing this nil value to the bellow function will panic as we access members of the cfg object.